packages/jest-cli/src/reporters/__tests__/summary_reporter.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
Ah, forgot about adding `@flow` pragma here. See how it's done in other files.

Reply:
I'll try it!

Reply:
Adding `@flow` made the test fail ... 

```
 39:   process.stderr.write = result => results.push(result);
                      ^^^^^ property `write`. Covariant property `write` incompatible with contravariant use in
```

And other *.test.js files also don't have `@flow` pragma. So is it possible to skip adding it here?

Reply:
Oh, forgot we only added it to integration tests. Cool, we can skip it for now.